Lib/ufo2ft/__init__.py

```diff
@@ -35,6 +36,7 @@
 
 
 logger = logging.getLogger(__name__)
+timer = Timer(logging.getLogger("fontmake.timer"), level=logging.INFO)
```

Comment:
I'd set this to level=logging.DEBUG -- this way we can enable --timing even without enabling all the debug messages which gets too noisy. we simply do (like fontmake currently does) by configuring for debug level the specific timing loggers, which by default emit debug messages.

If you keep this timer as level=INFO, then since e.g. fontmake defaults to INFO, they will always be visible by default, not only when --timing is enabled.

Does that make sense? Was it intentional?